snippets/capabilities-table.adoc

```diff
@@ -3,32 +3,65 @@
 The following table describes the `baselineCapabilitySet` values.
 
 .Cluster capabilities `baselineCapabilitySet` values description
-[cols=".^4,.^6a",options="header"]
+[cols=".^1,.^5a,.^4a",options="header"]
 |===
-|Value|Description
+|Value|Description|Default capabilities
 
 |`vCurrent`
-|Specify this option when you want to automatically add new, default capabilities that are introduced in new releases.
+|Specify this option when you want to automatically add the new, default capabilities that are introduced in new releases.
+|See the capabilities for each release.
 
 |`v4.11`
-|Specify this option when you want to enable the default capabilities for {product-title} 4.11. By specifying `v4.11`, capabilities that are introduced in newer versions of {product-title} are not enabled. The default capabilities in {product-title} 4.11 are `baremetal`, `MachineAPI`, `marketplace`, and `openshift-samples`.
+|Specify this option when you want to enable the default capabilities for {product-title} 4.11.
+By specifying `v4.11`, capabilities that are introduced in newer versions of {product-title} are not enabled.
+|* `baremetal`
+* `MachineAPI`
```

Comment:
The updated table looks good, except this special capability MachineAPI.
It is introduced on OCP4.14, and added into all baseline capability sets (including old cap set v4.11/v4.12/v4.13), that means capability MacineAPI is not in any cap set on OCP 4.13 and previous OCP version, but always in all cap set starting from OCP4.14.
So it is little confused if listing `MachineAPI` here.

Reply:
Oh, that's good to know! I think probably this can be addressed by careful cherrypicking (only port `MachineAPI` back to 4.14 docs), and it would be good to also add a note to explain this outlier situation.

I'll work on a note after 4.16 GA and hopefully get this version published fairly soon. Thanks for your review! :bow: